orchagent/fgnhgorch.cpp

```diff
@@ -1162,7 +1326,21 @@ bool FgNhgOrch::isRouteFineGrained(sai_object_id_t vrf_id, const IpPrefix &ipPre
                 }
             }
         }
+        if (static_fgnhg)
```

Comment:
do we need this logic? I think the logic above covers all the cases?

Reply:
Lets try to limit changes and logging here as much as possible, because this fn gets called from routeorch and is present on the path of all route creations. I would suggest keeping this function as-is with no changes, because it seems like it would work as-is.